velox/docs/develop/types.rst

```diff
@@ -209,6 +210,14 @@ As a result the IPPREFIX object stores *FFFF:FFFF::* and the length 32 for both
    IPPREFIX 'FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF/32' -- IPPREFIX 'FFFF:FFFF:0000:0000:0000:0000:0000:0000/32'
    IPPREFIX 'FFFF:FFFF:4455:6677:8899:AABB:CCDD:EEFF/32' -- IPPREFIX 'FFFF:FFFF:0000:0000:0000:0000:0000:0000/32'
 
+The BIGINTENUM type represents an enumerated value where the physical type is BIGINT.
+Each BIGINTENUM type carries an enum map, which is a mapping of string keys to BIGINT
+values. Each BIGINTENUM type has a unique name, ensuring that it can be distinctly
+identified within the system. Casting is permitted between the same enum types or between
```

Comment:
> Casting is permitted between the same enum types

What does that mean? Isn't it a no-op to cast a value of T to T?

Reply:
Yes, I wanted to stress that casting is not supported between different bigint enum types